### DISCUSSION TITLE
Height adjustment

### DISCUSSION THREAD

**Discussion by hasanshamsijeze** [vsg-dev/VulkanSceneGraph#1324] [Q&A]
How can I show earth maps with height?
for example, if there is a mountain, show the mountain's height.

**Comment by robertosfield** [comment]
In the VulkanSceneGraph/README.md you'll find a section on community projects, two of interest for whole earth terrain rendering are  Rocky and vsgCs:
----
Community projects:

[vsgSDL](https://github.com/ptrfun/vsgSDL) SDL integration with VulkanSceneGraph.
[vsgvr](https://github.com/geefr/vsgvr) OpenVR integration with VulkanSceneGraph.
[vsgCs](https://github.com/timoore/vsgCs) 3D Tiles and Cesium ion integration
[vsgEarth](https://github.com/timoore/vsgEarth) osgEarth integration
[rocky](https://github.com/pelicanmapping/rocky) 3D Geospatial Application Engine (Vulkan / C++17 / VSG). Pelican Mapping's successor to osgEerth.
